coredb-operator/src/defaults.rs

```diff
@@ -44,6 +46,6 @@ pub fn default_postgres_exporter_image() -> String {
     "quay.io/prometheuscommunity/postgres-exporter:v0.11.1".to_owned()
 }
 
-pub fn default_extensions() -> Vec<String> {
-    vec![]
+pub fn default_extensions() -> Vec<Extension> {
+    vec![Extension::default()]
```

Comment:
```suggestion
    vec![]
```
Same as above, I think we should default to empty `Vec<Extension>`.